refinedstorage2-platform-common/src/main/java/com/refinedmods/refinedstorage2/platform/common/tags/Tags.java

```diff
@@ -0,0 +1,26 @@
+package com.refinedmods.refinedstorage2.platform.common.tags;
+
+import net.minecraft.core.registries.Registries;
+import net.minecraft.tags.TagKey;
+import net.minecraft.world.item.Item;
+
+import static com.refinedmods.refinedstorage2.platform.common.util.IdentifierUtil.createIdentifier;
+
+public class Tags {
```

Comment:
```suggestion
public final class Tags {
```

(convention of the content classes)